regulations/views/preamble.py

```diff
@@ -178,12 +181,23 @@ def get(self, request, *args, **kwargs):
         if preamble is None:
             raise Http404
 
+        # @todo - right now we're shimming in fake data; eventually this data
```

Comment:
Looks like this logic needs to happen in `CFRChangesView` as well, since both views render both tocs.